app/index.html

```diff
@@ -1,282 +1,291 @@
-                                <a href="#"><i class="fa fa-facebook"></i></a>
-                                <a href="#"><i class="fa fa-google-plus"></i></a>
-                                <a href="#"><i class="fa fa-linkedin"></i></a>
-                            </li>
-                        </ul>
-                    </div>
-                    <!-- /.navbar-collapse -->
+                  <div class="navbar-header">
+                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <i class="fa fa-align-justify"></i>
+                    </button>
+                    <a class="navbar-brand" href="#">
+                      <h1>Zoo House</h1>
+                    </a>
+                  </div>
+                  <!-- Collect the nav links, forms, and other content for toggling -->
+                  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                    <ul class="nav navbar-nav navbar-right">
+                      <li class="active"><a href="#mission">mission</a></li>
+                      <li><a href="#about">what we do</a></li>
+                      <li><a href="#contact">contact</a></li>
+                      <li class="social-nav visible-lg">
+                        <a href="#"><i class="fa fa-twitter"></i></a>
+                        <a href="#"><i class="fa fa-facebook"></i></a>
+                        <a href="#"><i class="fa fa-google-plus"></i></a>
+                        <a href="#"><i class="fa fa-linkedin"></i></a>
+                      </li>
+                    </ul>
+                  </div>
+                  <!-- /.navbar-collapse -->
                 </div>
                 <!--/.container -->
-            </div>
-            <!--/.navbar -->
-
-        </header>
-
-        <section id="mission">
-            <div class="container">
+              </div>
+              <!--/.navbar -->
+            </header>
+            <section id="mission">
+              <div class="container">
                 <div class="row">
-                    <div class="col-md-8 col-md-push-2 clearfix">
-                        <div class="section-heading scrollpoint sp-effect3">
-                            <h1>.we are bond<span>y</span>
-                            </h1>
-                            <h4>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Inventore reiciendis vel reprehenderit expedita cupiditate repellat debitis! Est qui quae consectetur temporibus a illum deleniti? Quae, adipisci praesentium quis omnis dignissimos!</h4>
-                        </div>
+                  <div class="col-md-8 col-md-push-2 clearfix">
+                    <div class="section-heading scrollpoint sp-effect3">
+                      <h1>
+                        Welcome to Zoo House
```

Comment:
Danny said the 
`Welcome to Zoo House - We build awesome products with awesome people` should be on top of the video.